custom-connectors/Staffbase/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2691 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Staffbase",
+    "description": "This connector provides access to your Staffbase instance",
```

Comment:
@Nm-jens Can you please provide a detailed description value which any one can understand.

Documentation - [Description](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#description)